Crashlytics/Crashlytics/Handlers/FIRCLSMachException.c

```diff
@@ -287,12 +253,6 @@ static bool FIRCLSMachExceptionRegister(FIRCLSMachExceptionReadContext* context,
     return false;
   }
 
-  // Get the desired mask, which covers all the mach exceptions we are capable of handling,
-  // but clear out any that are in our ignore list.  We do this by ANDing with the bitwise
-  // negation.  Because we are only clearing bits, there's no way to set an incorrect mask
-  // using ignoreMask.
-  context->mask = FIRCLSMachExceptionMask() & ~ignoreMask;
```

Comment:
Is this property still in place on FIRCLSMachExceptionReadContext? We might need to set this

Reply:
Done